config_files/pipeline_low_latency_dash_config.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a sample pipeline configuration file for Shaka Streamer in live mode.
+# Here you configure resolutions, manifest formats, segment size, and more.
+
+# Streaming mode.  Can be live or vod.
+streaming_mode: live
+
+# A list of resolutions to encode.
+resolutions:
+  - 720p
+  - 480p
+
+# A list of channel layouts to encode.
+channel_layouts:
+  - stereo
+
+# The codecs to encode with.
+audio_codecs:
+  - aac
+video_codecs:
+  - h264
+
+# Manifest format must be DASH for LL-DASH streaming
+manifest_format:
+  - dash
+
+# Length of each segment in seconds.
+segment_size: 2
+
+# Availability window, or the number of seconds a segment remains available.
+availability_window: 300
+
+# Presentation delay, or how far back from the edge the player should be.
+presentation_delay: 30
```

Comment:
You may want a lower delay value for LL mode.  :-)

Reply:
Oops! Great catch!